src/components/Project/Extensions/Route53/index.js

```diff
@@ -196,14 +196,18 @@ export default class LoadBalancer extends React.Component {
     }
     
     var extraOptions = []
-    project.extensions.map(function(extension){
+    project.extensions.forEach(function(extension){
```

Comment:
Something like this uses the new ES6 styntax and may be a bit easier to maintain:

```
const { extensions } = project
const dns = extensions.find(ext => ext.key === 'dns')
const internalLoadbalancers = extensions.filter(ext => {
  return extension.extension.key.includes("loadbalancer")
    && extension.customConfig.type !== "internal"
})
const extraOptions = internalLoadbalancers.map(ext =>
  {
    key: ext.id,
    value: `${ext.extension.name} (${dns})`
  }
)
```